lib/BuildSystem/BuildFile.cpp

```diff
@@ -200,10 +225,11 @@ class OwnershipAnalysis {
   // This ensures TaskC will wait until TaskB is finished.
   void amendOutputOfOwnersWithConsumedSubpaths() {
     for (auto inputNode: allDirectoryInputNodes) {
-      ExternalCommand *owner = ownerOf(inputNode->getName());
+      Command *owner = includedOwnerOf(inputNode->getName());
       if (owner != nullptr) {
         auto ownerOutputs = owner->getOutputs();
-        if (std::find(ownerOutputs.begin(), ownerOutputs.end(), inputNode) == ownerOutputs.end()) {
+        auto condition = std::find(ownerOutputs.begin(), ownerOutputs.end(), inputNode) == ownerOutputs.end();
+        if (condition) {
```

Comment:
nit: don't need a separate `condition` variable.. I separated this into two statements during debugging because the debugger wasn't showing me the value of the condition when it was directly passed to `if` statement